fractal_server/app/routes/api/v2/task_collection.py

```diff
@@ -242,6 +253,73 @@ async def collect_tasks_pip(
     return state
 
 
+@router.post("/collect/custom/", status_code=201)
+async def collect_task_custom(
+    task_collect: TaskCollectCustomV2,
+    user: User = Depends(current_active_verified_user),
+    db: DBSyncSession = Depends(get_sync_db),
+) -> CollectionStateReadV2:
+
+    settings = Inject(get_settings)
+
+    if task_collect.package_root is None:
+        if task_collect.package_name is None:
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail=(
+                    "Must provide at least one of 'package_root' "
+                    "and 'package_name'."
+                ),
+            )
+        if settings.FRACTAL_RUNNER_BACKEND == "slurm_ssh":
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail="Cannot infer 'package_root' with 'slurm_ssh' backend.",
+            )
+        res = subprocess.run(  # nosec
+            shlex.split(
+                f"{task_collect.python_interpreter} "
+                f"-m pip show {task_collect.package_name}"
+            ),
+            capture_output=True,
+            encoding="utf8",
+        )
+        package_root = next(
+            it.split()[1]
```

Comment:
This is not yet `package_root`, see e.g.
```console
$ pip show devtools | grep Location
Location: /tmp/venv/lib/python3.10/site-packages
```

One needs to add the package name to the path obtained above.